ecl/hql/hqlfold.cpp

```diff
@@ -557,73 +557,84 @@ static bool isSimpleComparisonArg(IHqlExpression * expr)
  *       functions need to be declared with extern "C".
  *********************************************************/
  //MORE: This function never unloads the plugin dll - this may cause problems in the long run.
-IValue * foldExternalCall(IHqlExpression* expr, unsigned foldOptions, ITemplateContext *templateContext) 
```

Comment:
I had to split this function to avoid a compiler/glib bug on osx - the embedded asm was interfering with the ability to unwind stack frames on throw.